### PR TITLE
Add support for contact/testimonial forms

### PR DESCRIPTION
**Pull request by mattleff**
Depends on https://github.com/avorg/audioverse-graphql/pull/468 and fixes https://trello.com/c/gfbfmqUp/60-submit-testimony, https://trello.com/c/d42Dvd0f/171-contact-forms.

image

image

### CODE REVIEW

File: src/pages/[language]/contact/[type].ts [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
+	params,
+}: GetStaticPropsContext<{ language: string; type: string }>): Promise<
+	GetStaticPropsResult<ContactProps>
 > {
 	return {
-		props: {},
+		props: {
+			type: params?.type as string,
+		},
 		revalidate: REVALIDATE,
```
Comment (narthur) [inline src/pages/[language]/contact/[type].ts:23]: Does this need revalidate, or can we just rebuild this in CI and always serve static?

File: src/containers/contact.tsx [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+	useEffect(() => {
+		setRecipient(
+			{
+				general: PageContactRecipient.General,
+				support: PageContactRecipient.Technical,
+				testimonies: PageContactRecipient.Testimony,
+			}[type] || PageContactRecipient.General
+		);
+	}, [type]);
```
Comment (narthur) [inline src/containers/contact.tsx:42]: Good!

**Review by narthur — APPROVED** [review]
Great work!

File: src/pages/[language]/contact/[type].ts (hunk 1 above)
Comment (mattleff) [reply src/pages/[language]/contact/[type].ts]: Good point: 6fca93e

**Pull request merged**